src/txn/txn_ckpt.c

```diff
@@ -2456,6 +2456,8 @@ __wt_checkpoint_close(WT_SESSION_IMPL *session, bool final)
     if (!btree->modified && !bulk)
         return (__wt_evict_file(session, WT_SYNC_DISCARD));
 
+    WT_ASSERT(session, btree == NULL || !F_ISSET(btree, WT_BTREE_VERIFY));
```

Comment:
What is the purpose of this bit? Might be fine, I just don't understand why it's part of the PR.

Reply:
Sorry, I was testing something else and accidentally pushed it